processor_function/.env.sample

```diff
@@ -28,4 +28,9 @@ SEC_ARM_CLIENT_ID=__CHANGE_ME__
 SEC_ARM_CLIENT_SECRET=__CHANGE_ME__
 param_tfstate_resource_group_name=__CHANGE_ME__
 param_tfstate_container_name=__CHANGE_ME__
-param_tfstate_storage_account_name=__CHANGE_ME__
\ No newline at end of file
+param_tfstate_storage_account_name=__CHANGE_ME__
+
+# The below are needed for local development of the VMSS resource processor
+ARM_USE_MSI=false
+RESOURCE_PROCESSOR_CLIENT_ID=__CHANGE_ME__
```

Comment:
Is this really still needed for VMSS processor when working locally? I think we can use local login credentials to do even porter deployments as long as the user has the right set of permissions.

Reply:
Its the TF inside the bundles that needs them. As the bundles run inside a container, can't see how would access the local creds?